blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -221,6 +215,16 @@ public void trackResponse(HttpRequest request, HttpResponse response) {
 
     }
 
+    public void trackDelayedMetricsTenant(String tenantid, List<Metric> delayedMetrics) {
+        if (isTrackingDelayedMetrics) {
+            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metrics %s\n",tenantid);
+            for (Metric metric : delayedMetrics) {
+                logMessage += String.format("%s has collectionTime %d\n", metric.getLocator().toString(), metric.getCollectionTime());
```

Comment:
Do you think its worth printing the time in human readable format? Also the amount of delay.